frontend/src/modules/organization/components/view/organization-view-header.vue

```diff
@@ -187,13 +138,22 @@ const props = defineProps({
 });
 
 const showMore = ref(false);
+const query = ref('');
+const allMembers = reactive([]);
+const limit = ref(20);
+const offset = ref(0);
+const store = useStore();
```

Comment:
We dont use store like this anymore
We have helpers in vuex.helpers
const {currentTenant} = mapGetters('auth')